usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_transaction.md

```diff
@@ -0,0 +1,82 @@
+## [Spending by Transaction](#spending-by-transaction)
+**Route:** `/api/v2/search/spending_by_transaction/`
+
+**Method:** `POST`
+
+This route takes keyword search fields, and returns the fields of the searched term.
+
+### Request
+field: Defines what award variables are returned.
+
+filter: Defines how the transactions are filtered.  Search term and pannel-name are required. Transaction type is one of pane they are looking at. Can be one of the following- ["Contracts", "Grants", "Direct Payments", "Loans", "Other"]
```

Comment:
The parameter is `filters`. The explanation is wrong. Just use this:

filters: An object with `keyword` and `award_type_codes` keys. `keyword` should be a string that you are performing a keyword search operation with. `award_type_codes` is an array of strings of the award type codes that should be searched within.

A list of award type codes can be found at http://fedspendingtransparency.github.io/whitepapers/types/

Reply:
done